src/main/java/com/team1/dealerApp/user/UserMapper.java

```diff
@@ -23,7 +22,6 @@ public UserDTO toUserDTO(User user) {
                 .lastName(user.getLastName())
                 .email(user.getEmail())
                 .phoneNumber(user.getPhoneNumber())
```

Comment:
Si potrebbe mettere anche .role(user.getRole()) per passare il ruolo dell'utente al DTO in uscita